applications/asset_tracker_v2/doc/gnss_module.rst

```diff
@@ -41,8 +41,9 @@ LNA configuration
 =================
 
 Different devices have different antenna and LNA setups depending on the antenna type (onboard or external).
-The module has Kconfig options to control the LNA using either the COEX0 or MAGPIO interface of the nRF9160.
-See :ref:`configuration_options` for more details on how to configure the antenna and LNA.
+The application uses the :ref:`lib_modem_antenna` library for configuring the LNA.
+The Modem antenna library has Kconfig options to control the LNA using either the COEX0 or MAGPIO interface of the nRF9160.
+See the library documentation for more details on how to configure the antenna and LNA.
```

Comment:
```suggestion
The library has Kconfig options to control the LNA using either the COEX0 or MAGPIO interface of the nRF9160.
See the library documentation for more details on how to configure the antenna and LNA.
```
I believe we are fine to refer to the "library" only, as we just mentioned it in the previous sentence.